level_file_temp/setup.spec

```diff
@@ -0,0 +1,9 @@
+1 (tag : <tag_name>)
+2 : 1 (<branch_name>)
+3
+4 : 1 (Example_child_of_1)
+5 (tag : Example_child_of_4)
+6
+7 : 3 6 (Example_merge_node_of_3_&_4)
```

Comment:
If we use &, then we would have to always put the tag name in quotes because & is treated as a special operator, so it might be better to use `merge-3-and-4` or something else

Reply:
done